project_setup/scripts/terraform-to-secrets

```diff
@@ -44,6 +44,7 @@ Options:
   -l --log-level=LEVEL   If specified, then the log level will be set to
                          the specified value.  Valid values are "debug", "info",
                          "warning", "error", and "critical". [default: info]
+  -m --remote-state      Look for secrets in remote state resources also.
```

Comment:
Do we want to give this a default value of `False`?

Reply:
If the `--remote-state` flag is provided, the value is set to true, otherwise it's false.  I don't see a benefit to including a default value here, but if you have a good reason, please let us know.

Reply:
Ah, right. Sorry I've been using `argparse` more lately and forgot about that detail with how `docopt` parses things.